src/views/Restoration/RestorationBusinessInformation.vue

```diff
@@ -111,31 +137,24 @@ export default class RestorationBusinessInformation extends Mixins(CommonMixin)
   }
 
   get isEmptyRecordsAddress () : boolean {
-    if (!this.addresses.recordsOffice?.mailingAddress || !this.addresses.recordsOffice?.deliveryAddress) {
```

Comment:
This old code doesn't work because we always initialize the addresses object in the store.